Client/Frontend/Browser/Toolbars/BottomToolbar/Menu/Bookmarks/AddEditBookmarkTableViewController.swift

```diff
@@ -317,6 +355,18 @@ class AddEditBookmarkTableViewController: UITableViewController {
             dismiss(animated: true)
         }
     }
+    
+    private func addListOfBookmarks(_ tabs: [Tab], parentFolder: Bookmarkv2? = nil) {
+        isLoading = true
+        
+        for tab in tabs {
+            if let url = tab.url, url.isWebPage(), !url.isAboutHomeURL {
+                Bookmarkv2.add(url: url, title: tab.title, parentFolder: parentFolder)
```

Comment:
is `Saved Tabs` folder created once or each time bookmarks are saved?

I mean you have 10 bookmarks, you save them,
then another browsing session you add 5 bookmarks, are they saved to a single folder?

Reply:
I followed the pattern Safari uses here. I think it makes sense, every-time user wanted to bookmark all tabs, create folder screen is presented with some default name ("Saved Tabs"). User can change the folder name, choose another parent folder etc. and this folder is created. 
After the folder is created all the valid information that  can be added as bookmarks from tab tray (eliminating empty tabs etc) will be added as bookmark items in this folder.